matador/plotting/ir_plotting.py

```diff
@@ -0,0 +1,126 @@
+""" This submodule implements functions useful for plotting
+the results of infrared and Raman spectroscopy calculations.
+
+"""
+
+from matador.plotting.plotting import plotting_function
+
+
+def read_ir_file(seed):
+    """ This function should take the seed filename and
+    read in the IR data, probably into a dictionary.
+
+    Parameters:
+        seed (str): the filename to read.
+
+    Returns:
+        dict: containing IR data
+
+    """
+
+    with open(seed + '.phonon', 'r') as f:
+        flines = f.readlines()
+    for ind, line in enumerate(flines):
+        if 'Number of branches' in line:
+            no_branches = int(line.split()[3])
+        if 'END header' in line:
+            header_end = ind
+
+    # contains IR and Raman data
+    ir_dat = dict()
+    for line in flines[header_end+2:header_end+no_branches+2]:
+        line_split = line.split()
+        if len(line_split) < 2:
+            print_failure('Failed to read IR data. Check .phonon file.')
+            exit()
+        wavenumber = float(line_split[1])
+        ir = float(line_split[2])
+        # checks for raman data
+        if len(line_split) > 3:
+            raman = float(line_split[3])
+            if wavenumber in ir_dat:
+                ir_dat[wavenumber] = [ir_dat[wavenumber][0]+ir, ir_dat[wavenumber][1]+raman]
+            else:
+                ir_dat[wavenumber] = [ir, raman]
+        else:
+            if wavenumber in ir_dat:
+                ir_dat[wavenumber] = [ir_dat[wavenumber][0]+ir]
+            else:
+                ir_dat[wavenumber] = [ir]
+
+    return(ir_dat)
+
+# this decorator will automatically set the style of the plot
+# to match the user's config/matador defaults, and safely handles
+# e.g. X-forwarding and file writing
+@plotting_function
+def plot_ir_spectrum(seed, ir_ss, ir_bs):
+    """ This function plots the IR spectrum found in the given file.
+
+        Parameters:
+        bin_width: The width of the wavenumber bin in cm^-1. Determined by step_size and bin_scaler
```

Comment:
Sphinx will parse the type you provide in docstrings

```suggestion
        bin_width (float): The width of the wavenumber bin in cm^-1. Determined by step_size and bin_scaler
```

Reply:
A note on docstrings generally, in the convention we're using for matador (Google-style here https://www.sphinx-doc.org/en/master/usage/extensions/napoleon.html). The Parameters heading is used to list all arguments to the function and Keyword Arguments is used to list all the Python keyword arguments (optional keywords with default values).